modules/installation-creating-aws-control-plane.adoc

```diff
@@ -0,0 +1,211 @@
+your bootstrap node.
+See link:https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html[How Do I Create an S3 Bucket?]
+in the AWS documentation.
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the networking objects and load balancers and save it to your computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "RhcosAmi", <3>
+    "ParameterValue": "ami-07f604c5f18a1e7a9" <4>
+  },
+  {
+    "ParameterKey": "AutoRegisterDNS", <5>
+    "ParameterValue": "yes" <6>
+  },
+  {
+    "ParameterKey": "PrivateHostedZoneId", <7>
+    "ParameterValue": "1" <8>
+  },
+  {
+    "ParameterKey": "PrivateHostedZoneName", <9>
+    "ParameterValue": "example.com" <10>
+  },
+  {
+    "ParameterKey": "Master0Subnet", <11>
+    "ParameterValue": "subnet-0dfe1c9412a54d352" <12>
+  },
+  {
+    "ParameterKey": "Master1Subnet", <13>
+    "ParameterValue": "subnet-0dfe1c9412a54d352" <14>
+  },
+  {
+    "ParameterKey": "Master2Subnet", <15>
+    "ParameterValue": "subnet-0dfe1c9412a54d352" <16>
+  },
+  {
+    "ParameterKey": "MasterSecurityGroupId", <17>
+    "ParameterValue": "sg-0a53cd1337f15u2ee" <18>
+  },
+  {
+    "ParameterKey": "IgnitionLocation", <19>
+    "ParameterValue": "https://api.<cluster_name>.<domain_name>:22623/config/master" <20>
+  },
+  {
+    "ParameterKey": "CertificateAuthorities", <21>
+    "ParameterValue": "data:text/plain;charset=utf-8;base64,ABC...xYz==" <22>
+  },
+  {
+    "ParameterKey": "MasterInstanceProfileName", <23>
+    "ParameterValue": "<roles_stack>-MasterInstanceProfile-1HCV65K4PMWCMN" <24>
+  },
+  {
+    "ParameterKey": "MasterInstanceType", <25>
+    "ParameterValue": "m4.xlarge" <26>
+  },
+  {
+    "ParameterKey": "AutoRegisterELB", <27>
+    "ParameterValue": "yes" <28>
+  },
+  {
+    "ParameterKey": "RegisterNlbIpTargetsLambdaArn", <29>
+    "ParameterValue": "arn:aws:lambda:<zone>:269733483066:function:<dns_stack>-RegisterNlbIpTargets-NHEKB9IHN5XZ" <30>
+  },
+  {
+    "ParameterKey": "ExternalApiTargetGroupArn", <31>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Exter-D374J8IQ3K03/b26760f600ff322e" <32>
+  },
+  {
+    "ParameterKey": "InternalApiTargetGroupArn", <33>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Inter-1Q3IAOSC8ZNNM/8e85701acf15632d" <34>
+  },
+  {
+    "ParameterKey": "InternalServiceTargetGroupArn", <35>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Inter-1INRIWR8L3627/4cd9e764bb418f47" <36>
+  }
+]
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the
```

Comment:
cluster name is also used when user decide to generate only ignition-configs files.